flink-connectors/flink-connector-elasticsearch6/src/main/java/org/apache/flink/streaming/connectors/elasticsearch/table/Elasticsearch6DynamicSinkFactory.java

```diff
@@ -113,7 +113,7 @@ private void validate(Elasticsearch6Configuration config, Configuration original
 		validate(
 			maxActions == -1 || maxActions >= 1,
 			() -> String.format(
-				"'%s' must be at least 1 character. Got: %s",
+				"'%s' must be at least 1. Got: %s",
```

Comment:
cc @dawidwys , is this a typo?

Reply:
@wuchong @dawidwys ,it is a typo indeed. Because sink.bulk-flush.max-actions config is int type. And you can compare with other in this class

Reply:
Sorry for a late reply. Yes, it is a typo.